server/README.md

```diff
@@ -1,4 +1,4 @@
-[![ci-server](https://github.com/reearth/reearth/actions/workflows/ci_server.yml/badge.svg)](https://github.com/reearth/reearth/actions/workflows/ci_server.yml) [![codecov](https://codecov.io/gh/reearth/reearth/branch/main/graph/badge.svg?flag=server)](https://codecov.io/gh/reearth/reearth)
+[![ci-server](https://github.com/reearth/reearth-visualizer-visualizer-visualizer/actions/workflows/ci_server.yml/badge.svg)](https://github.com/reearth/reearth-visualizer/actions/workflows/ci_server.yml) [![codecov](https://codecov.io/gh/reearth/reearth/branch/main/graph/badge.svg?flag=server)](https://codecov.io/gh/reearth/reearth)
```

Comment:
Sorry is this Url `https://github.com/reearth/reearth-visualizer-visualizer-visualizer/actions/workflows/ci_server.yml/badge.svg `correct?

Reply:
fixed 🙏🏼
https://github.com/reearth/reearth-visualizer/pull/995/commits/afc5b5bcc28bb3a1533289af6690c4d694505fe1

Reply:
Ahh still `reearth-visualizer-visualizer` two visualizer needed?